common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -417,6 +417,12 @@ impl StateMachine {
                 ref db_name,
                 ref table_name,
             } => {
+                if db_name == "system" {
```

Comment:
Em, we should throw the error in query/catalog: https://github.com/datafuselabs/databend/blob/9c2e1d606688bbac5134eaaee33dfb3b798dfb8d/query/src/catalogs/impls/catalog/system_catalog.rs#L158

But it doesn't, it need to address why

Reply:
fixed :)